ftp/gameap-files/uninstall-files-windows.ps1

```
@@ -188,9 +195,17 @@ if (Test-Path -LiteralPath $LogDir) {
 }
 
 if ($Purge) {
+    # The configuration directory sits inside the data directory; only that
+    # directory goes, never the data directory or a drive root.
+    $trimmedConfigDir = $ConfigDir.TrimEnd("\", "/")
+    if (($DataDir -and $trimmedConfigDir -eq $DataDir.TrimEnd("\", "/")) -or
+        $trimmedConfigDir -eq [IO.Path]::GetPathRoot($ConfigDir).TrimEnd("\", "/")) {
+        throw "Refusing to purge $ConfigDir: it is the data directory or a drive root."
```

PSScriptAnalyzer (1.25.0)

[error] Variable reference is not valid. ':' was not followed by a valid variable name character. Consider using ${} to delimit the name.

(InvalidVariableReferenceWithDrive)